### ISSUE TITLE
Error: Job process error, what: something's wrong: active_toms is empty

### ISSUE THREAD

**Issue by sbakiu** [kineticadb/kinetica-api-python#4]
Hi,
I am using release v6.1.0.1 and I receive the following error when executing `flush()` 
```
File "/python/gpudb/gpudb_ingestor.py", line 2035, in insert_record
    self.__flush( queue, worker_queue.get_gpudb() )
  File "/python/gpudb/gpudb_ingestor.py", line 2122, in __flush
    raise InsertionException( str(e), queue )
gpudb_ingestor.InsertionException: Error: 'Job process error, what:
 'something's wrong: active_toms is empty. (E/Ec:270); code:1 'Error'''
```
What can be the reason for this error?

**Comment by mmahmud** [comment]
Hi Sadik,

Do you have access to the Kinetica log?  It would be useful to debug the
scenario; the last 10k or so lines would be great to have.

Thanks!

**Comment by sbakiu** [comment]
Hi @mmahmud 
Sadly I cannot post that long comment here. Here is an extract:
```
2018-02-21 07:33:13.394 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:399 - Regenerating foreign key - primary key lookups took: 0.00001 s
2018-02-21 07:33:13.394 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:339 - Done loading persist for TOM: 3
2018-02-21 07:33:13.394 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:353 - Persist loaded
2018-02-21 07:33:13.394 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:51 - HTTP starting server on port:9216
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:98 - Initializing Non-HTTPS
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:108 - Gaia is running, HTTP server listening on port: 9216
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 24000
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 24001
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 24002
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 24003
2018-02-21 07:33:13.395 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal GlobalRegistry/GlobalRegistry.cpp:1124 - send_member_update gaia::rank_info_status::rank_info_status_ gaia::rank_info::*, value: 'running', obj: {version:10,rank:25,rank_id:'25 : 10.0.1.179 : 3219',rank_mode:run,rank_status:running,app_version:'6.1.0.3.20180205222007',pid:3219,start_time:1519198358,start_time_str:'2018-02-21 07:32:38',hostname:'ip-10-0-1-179.eu-west-1.compute.internal',host_id:'10.0.1.179',gpu_ids:1,gpu_index:0}
2018-02-21 07:33:13.396 INFO  (3219,3219,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:406 - Manager for rank: 25 running...
2018-02-21 07:33:13.450 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:375 - TypeObjectManagerFileSync::loadGaiaSets gaiaSets.size=3 in /data_kinetica/gpudb/rank-30/tom-3 time: 6.55610 s
2018-02-21 07:33:13.450 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:399 - Regenerating foreign key - primary key lookups took: 0.00002 s
2018-02-21 07:33:13.450 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:339 - Done loading persist for TOM: 3
2018-02-21 07:33:13.450 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:353 - Persist loaded
2018-02-21 07:33:13.450 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:51 - HTTP starting server on port:9221
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:98 - Initializing Non-HTTPS
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:108 - Gaia is running, HTTP server listening on port: 9221
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 29000
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 29001
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 29002
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 29003
2018-02-21 07:33:13.451 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal GlobalRegistry/GlobalRegistry.cpp:1124 - send_member_update gaia::rank_info_status::rank_info_status_ gaia::rank_info::*, value: 'running', obj: {version:10,rank:30,rank_id:'30 : 10.0.1.179 : 3380',rank_mode:run,rank_status:running,app_version:'6.1.0.3.20180205222007',pid:3380,start_time:1519198358,start_time_str:'2018-02-21 07:32:38',hostname:'ip-10-0-1-179.eu-west-1.compute.internal',host_id:'10.0.1.179',gpu_ids:1,gpu_index:0}
2018-02-21 07:33:13.452 INFO  (3380,3380,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:406 - Manager for rank: 30 running...
2018-02-21 07:33:13.888 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:375 - TypeObjectManagerFileSync::loadGaiaSets gaiaSets.size=3 in /data_kinetica/gpudb/rank-26/tom-3 time: 5.69191 s
2018-02-21 07:33:13.888 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:399 - Regenerating foreign key - primary key lookups took: 0.00001 s
2018-02-21 07:33:13.888 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:339 - Done loading persist for TOM: 3
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:353 - Persist loaded
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:51 - HTTP starting server on port:9217
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:98 - Initializing Non-HTTPS
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:108 - Gaia is running, HTTP server listening on port: 9217
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 25000
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 25001
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 25002
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 25003
2018-02-21 07:33:13.889 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal GlobalRegistry/GlobalRegistry.cpp:1124 - send_member_update gaia::rank_info_status::rank_info_status_ gaia::rank_info::*, value: 'running', obj: {version:10,rank:26,rank_id:'26 : 10.0.1.179 : 3275',rank_mode:run,rank_status:running,app_version:'6.1.0.3.20180205222007',pid:3275,start_time:1519198358,start_time_str:'2018-02-21 07:32:38',hostname:'ip-10-0-1-179.eu-west-1.compute.internal',host_id:'10.0.1.179',gpu_ids:1,gpu_index:0}
2018-02-21 07:33:13.891 INFO  (3275,3275,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:406 - Manager for rank: 26 running...
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:375 - TypeObjectManagerFileSync::loadGaiaSets gaiaSets.size=3 in /data_kinetica/gpudb/rank-28/tom-3 time: 5.07083 s
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:399 - Regenerating foreign key - primary key lookups took: 0.00001 s
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:339 - Done loading persist for TOM: 3
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:353 - Persist loaded
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:51 - HTTP starting server on port:9219
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:98 - Initializing Non-HTTPS
2018-02-21 07:33:13.995 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:108 - Gaia is running, HTTP server listening on port: 9219
2018-02-21 07:33:13.996 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 27000
2018-02-21 07:33:13.996 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 27001
2018-02-21 07:33:13.996 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 27002
2018-02-21 07:33:13.996 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 27003
2018-02-21 07:33:13.996 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal GlobalRegistry/GlobalRegistry.cpp:1124 - send_member_update gaia::rank_info_status::rank_info_status_ gaia::rank_info::*, value: 'running', obj: {version:10,rank:28,rank_id:'28 : 10.0.1.179 : 3340',rank_mode:run,rank_status:running,app_version:'6.1.0.3.20180205222007',pid:3340,start_time:1519198359,start_time_str:'2018-02-21 07:32:39',hostname:'ip-10-0-1-179.eu-west-1.compute.internal',host_id:'10.0.1.179',gpu_ids:1,gpu_index:0}
2018-02-21 07:33:13.997 INFO  (3340,3340,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:406 - Manager for rank: 28 running...
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:375 - TypeObjectManagerFileSync::loadGaiaSets gaiaSets.size=3 in /data_kinetica/gpudb/rank-27/tom-3 time: 5.02608 s
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal TypeManagement/TypeObjectManager.cpp:399 - Regenerating foreign key - primary key lookups took: 0.00001 s
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:339 - Done loading persist for TOM: 3
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:353 - Persist loaded
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:51 - HTTP starting server on port:9218
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:98 - Initializing Non-HTTPS
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal HttpServer/GaiaHttpServer.cpp:108 - Gaia is running, HTTP server listening on port: 9218
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 26000
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 26001
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 26002
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal ProcManager/ProcExecutionManager.cpp:19 - Creating proc execution manager for TOM 26003
2018-02-21 07:33:14.014 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal GlobalRegistry/GlobalRegistry.cpp:1124 - send_member_update gaia::rank_info_status::rank_info_status_ gaia::rank_info::*, value: 'running', obj: {version:10,rank:27,rank_id:'27 : 10.0.1.179 : 3321',rank_mode:run,rank_status:running,app_version:'6.1.0.3.20180205222007',pid:3321,start_time:1519198358,start_time_str:'2018-02-21 07:32:38',hostname:'ip-10-0-1-179.eu-west-1.compute.internal',host_id:'10.0.1.179',gpu_ids:1,gpu_index:0}
2018-02-21 07:33:14.015 INFO  (3321,3321,gpudb_cluster_c   ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:406 - Manager for rank: 27 running...
2018-02-21 07:33:16.916 INFO  (3321,3698,r27/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=27, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3321,3698,r27/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3340,3703,r28/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=28, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3340,3703,r28/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3219,3690,r25/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=25, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3219,3690,r25/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3387,3712,r31/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=31, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3387,3712,r31/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3275,3695,r26/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=26, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3275,3695,r26/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3373,3706,r29/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=29, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3373,3706,r29/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3380,3709,r30/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=30, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3380,3709,r30/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:33:16.916 INFO  (3389,3715,r32/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:528 - Manager::setShardInfo rank=32, tom_count=4, shardInfo.active_toms[CommunicatorUtils::getRank()].size()=0
2018-02-21 07:33:16.916 INFO  (3389,3715,r32/gpudb_gblreg  ) ip-10-0-1-179.eu-west-1.compute.internal Manager.cpp:568 - Manager::setShardInfo no backup toms info
2018-02-21 07:44:14.938 INFO  (3219,4947,r25/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/EndpointHttpRequestHandler.cpp:102 - JobId:1; Request URI: /insert/records from: 10.0.1.207 type: application/octet-stream user: anonymous
2018-02-21 07:44:14.953 UERR  (3219,4947,r25/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/Endpoint_insert_records.cpp:270 - JobId:1.1; 'something's wrong: active_toms is empty. (E/Ec:270)'
2018-02-21 07:44:14.953 INFO  (3219,4947,r25/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/EndpointHttpRequestHandler.cpp:140 - JobId:1; Request URI /insert/records completed in 0.01737 s bytes: 768410
2018-02-21 07:44:41.214 INFO  (3373,4941,r29/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/EndpointHttpRequestHandler.cpp:102 - JobId:1; Request URI: /insert/records from: 10.0.1.207 type: application/octet-stream user: anonymous
2018-02-21 07:44:41.230 UERR  (3373,4941,r29/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/Endpoint_insert_records.cpp:270 - JobId:1.1; 'something's wrong: active_toms is empty. (E/Ec:270)'
2018-02-21 07:44:41.230 INFO  (3373,4941,r29/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/EndpointHttpRequestHandler.cpp:140 - JobId:1; Request URI /insert/records completed in 0.01858 s bytes: 768837
2018-02-21 07:46:30.929 INFO  (3380,4950,r30/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/EndpointHttpRequestHandler.cpp:102 - JobId:1; Request URI: /insert/records from: 10.0.1.207 type: application/octet-stream user: anonymous
2018-02-21 07:46:30.933 UERR  (3380,4950,r30/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/Endpoint_insert_records.cpp:270 - JobId:1.1; 'something's wrong: active_toms is empty. (E/Ec:270)'
2018-02-21 07:46:30.934 INFO  (3380,4950,r30/gpudb_cluster_c) ip-10-0-1-179.eu-west-1.compute.internal Endpoint/EndpointHttpRequestHandler.cpp:140 - JobId:1; Request URI /insert/records completed in 0.00608 s bytes: 497724
```

**Comment by ghost** [comment]
Hi Sadik,

Could you try to restart Kinetica services before attempting the ingestion again?
If you did, were you able to reproduce after that?

**Comment by sbakiu** [comment]
Hi Franz,
I did and I still get the same issue

**Comment by ghost** [comment]
Hi Sadik,
I suspect we might need to clear the persist and reingest from scratch. Is that something that you would consider doing?

If you do, to clear it follow the steps below:
- Shut down Kinetica
- If your persist directory is the default then remove the folder /opt/gpudb/persist/gpudb (rm -rf /opt/gpudb/persist/gpudb) If you changed the persist directory, then remove the folder called gpudb inside the directory created.
- Restart Kinetica
- Try the ingestion again

I would also recommend upgrading to the latest GA version available (GA5 6.1.0.5) to take advantage of our latest fixes.

**Comment by sbakiu** [comment]
Hi Franz,

Thanks for the support. I cleared the persist folder and it worked.
